integrated_channels/canvas/client.py

```diff
@@ -117,8 +117,70 @@ def delete_content_metadata(self, serialized_data):
 
     def create_assessment_reporting(self, user_id, payload):
         """
-        Not implemented yet
+        Send assessment level learner data, retrieved by the integrated channels exporter, to Canvas in the form of
+        an assignment and submission.
         """
+        learner_data = json.loads(payload)
+        self._create_session()
+
+        # Retrieve the Canvas user ID from the user's edx email (it is assumed that the learner's Edx
+        # and Canvas emails will match).
+        canvas_user_id = self._search_for_canvas_user_by_email(user_id)
+
+        # With the Canvas user ID, retrieve all courses for the user.
+        user_courses = self._get_canvas_user_courses_by_id(canvas_user_id)
+
+        # Find the course who's integration ID matches the learner data course ID
+        course_id = None
+        for course in user_courses:
+            integration_id = course['integration_id']
+            if integration_id == learner_data['courseID']:
```

Comment:
for understanding:
I am guessing we (=the learner data export) only send 1 course at a time? (I ask because otherwise the structure here and particularly the break wouldn't work if we send more than one at a time in the payload)

Reply:
yeah, the exports are handled by enrollment, so it's one at a time.